src/brod_consumer.erl

```diff
@@ -48,238 +42,209 @@
         , handle_info/2
         , terminate/2
         , code_change/3
-        , format_status/2
         ]).
 
 -include("brod.hrl").
 -include("brod_int.hrl").
 
--record(state, { hosts         :: [{string(), integer()}]
-               , socket        :: #socket{}
+-type consumer_option() :: offset
```

Comment:
we have used the name 'config' for clients and producer.
i.e. client_config and producer_config.
perhaps consumer_config here ?